spec/system/trn_requests_spec.rb

```diff
@@ -598,6 +599,26 @@ def given_the_use_dqt_api_itt_providers_feature_is_enabled
     FeatureFlag.activate(:use_dqt_api_itt_providers)
   end
 
+  def given_the_zendesk_integration_feature_is_enabled
+    FeatureFlag.activate(:zendesk_integration)
+  end
+
+  def given_the_match_on_email_feature_is_enabled
+    FeatureFlag.activate(:match_on_email)
+  end
+
+  def and_the_match_on_email_feature_is_disabled
+    FeatureFlag.deactivate(:match_on_email)
+  end
+
+  def given_the_ask_for_trn_feature_is_enabled
+    FeatureFlag.activate(:ask_for_trn)
+  end
+
+  def given_the_ask_for_trn_feature_is_disabled
+    FeatureFlag.deactivate(:ask_for_trn)
+  end
```

Comment:
Are these being used?

Reply:
It was to be used for the system specs which i am working on

Reply:
No they are not. Taking them out